browser/permissions/permission_lifetime_manager_browsertest.cc

```diff
@@ -55,17 +56,27 @@ const char kPreTestDataFileName[] = "pre_test_data";
 class PermissionLifetimeManagerBrowserTest : public InProcessBrowserTest {
  public:
   PermissionLifetimeManagerBrowserTest()
-      : https_server_(net::EmbeddedTestServer::TYPE_HTTPS) {
-  }
+      : https_server_(net::EmbeddedTestServer::TYPE_HTTPS) {}
 
   ~PermissionLifetimeManagerBrowserTest() override = default;
 
   void SetUpCommandLine(base::CommandLine* command_line) override {
     InProcessBrowserTest::SetUpCommandLine(command_line);
-    command_line->AppendSwitch(switches::kIgnoreCertificateErrors);
+    mock_cert_verifier_.SetUpCommandLine(command_line);
+  }
+
+  void SetUpInProcessBrowserTestFixture() override {
+    InProcessBrowserTest::SetUpInProcessBrowserTestFixture();
+    mock_cert_verifier_.SetUpInProcessBrowserTestFixture();
+  }
+
+  void TearDownInProcessBrowserTestFixture() override {
+    InProcessBrowserTest::TearDownInProcessBrowserTestFixture();
+    mock_cert_verifier_.TearDownInProcessBrowserTestFixture();
   }
 
   void SetUpOnMainThread() override {
```

Comment:
let's call parent class here to be safe: `InProcessBrowserTest::SetUpOnMainThread();`

Reply:
added in https://github.com/brave/brave-core/commit/52c05a3594dbb439663b7b100cc719d80f3cb7db